src/test/java/trello/steps/ListSteps.java

```diff
@@ -70,6 +69,65 @@ public void createNewList(final String listTitle) {
     @Then("I should see the new created list with the given title")
     public void verifyList() {
         boardPage = new BoardPage();
-        Assert.assertTrue(boardPage.isThereThisListByTitle(context.getList().getTitle()));
+        Assert.assertTrue(boardPage.isThereThisListByTitle(context.getListSource().getTitle()));
+    }
+
+    /**
+     * Moves all cards in given list to given target list.
+     *
+     * @param listSource is the list which cards are wanted to move from.
+     * @param listTarget is the list which the cards are wanted to move to. This list must be empty.
+     */
+    @When("I move all cards in (.*) to (.*)")
+    public void moveAllCardsInList(final String listSource, final String listTarget) {
+        boardPage = new BoardPage();
+        context.getListSource().setTitle(listSource);
+        context.getListSource().setCards(boardPage.getCardsInList(listSource));
+        context.getListTarget().setTitle(listTarget);
```

Comment:
I think you don need this method in Context class only you have to use only object List in this steps.
E.G.
 List listSource = new List(name); 
 List lisTarget = new List();
Maybe if you need share this object to delete more later you need a map or list into context.
E.G
Map<String, List> mapOfList;
with your gets.

Reply:
done